torchvision/prototype/transforms/functional/_meta.py

```diff
@@ -61,6 +61,15 @@ def convert_bounding_box_format(
     return bounding_box
 
 
+def clamp_bounding_box(
```

Comment:
Can we maybe also make this a method on `features.BoundingBox`? I'm aware that `clamp` is already used and so is `clip`. Maybe we can call it `trim`?

Reply:
We may try to override existing `Tensor.clamp` ?

Reply:
> We may try to override existing Tensor.clamp ?

I would avoid overwriting default tensor methods. Even deprecated ones might be a problem.

Reply:
Why do you think it could be harmful ?

Reply:
I know we do this more on the resize which is deprecated but as we discussed in that case it might surprise users. We should probably ask for feedback from the community on the topic. From one hand, subclassing Tensor means we can overwrite methods to make the more "relevant" to the specific type. On the other hand, it might confuse users who expect that they are accessing the parent method. To avoid blocking this PR, I think we can go ahead without a method on the BBox object and raise this as an open question on the API design.